international-labour-organization.csl

```diff
@@ -16,16 +16,19 @@
     <author>
       <name>Vesa Sivunen</name>
     </author>
+    <author>
+      <name>Stephane Givkovic</name>
+    </author>
     <category citation-format="author-date"/>
     <category field="generic-base"/>
     <summary>The Harvard author-date style adapted for ILO house style</summary>
-    <updated>2013-01-21T15:37:19+00:00</updated>
+    <updated>2017-03-14T08:22:32+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="editor">
-    <names variable="editor" delimiter=", ">
+    <names variable="editor" delimiter=",">
       <name name-as-sort-order="all" and="text" sort-separator=", " initialize-with=". " delimiter=", "/>
-      <label form="short" prefix=" "/>
+      <label form="short"/>
```

Comment:
this seems wrong to me? Do you really want <code>Smith, A.ed.</code> without any space?